sessionctx/varsutil/varsutil.go

```diff
@@ -107,10 +144,9 @@ func SetSessionSystemVar(vars *variable.SessionVars, name string, value types.Da
 			vars.StrictSQLMode = false
 		}
 		// Modes is a list of different modes separated by commas.
-		modes := strings.Split(sVal, ",")
-		var sqlMode mysql.SQLMode
-		for _, mode := range modes {
-			sqlMode = sqlMode | mysql.GetSQLMode(mode)
+		sqlMode, err2 := mysql.GetSQLMode(sVal)
```

Comment:
s/err2/err/

Reply:
use the name `err` would cover outside definition of `err` in line `sVal, err := value.ToString()`

Reply:
it does not matter, if outside err is not nil, function will be returned before this instruction, we can safely override it

Reply:
But the compile-time check would fail when execute `make dev`.....

Reply:
I see.. let's keep it